Source/DotSpatial.Data/AttributeTable.cs

```diff
@@ -888,6 +892,34 @@ public void Fill(int numRows)
         }
 
         /// <summary>
+        /// Exports the dbf file content to a stream.
+        /// </summary>
+        /// <returns>A stream that contains the dbf file content.</returns>
+        public Stream ExportDbfToStream()
+        {
+            MemoryStream dbfStream = new MemoryStream();
+            UpdateSchema();
+
+            try
+            {
+                using (var inMemoryStream = new MemoryStream())
+                using (_writer = new BinaryWriter(inMemoryStream))

```

Comment:
Ah, ok. I think later we should refactor this class, because this `_writer` initialized in several places, and logic is messy,